src/qt/trafficgraphwidget.cpp

```diff
@@ -7,6 +7,7 @@
 #include <qt/clientmodel.h>
 
 #include <QPainter>
+#include <QPainterPath>
```

Comment:
Is this something that upstream also added?

Reply:
Yes. It is needed to compile with recent versions of qt. (And even upstream, all versions of Core from 0.18 and before fail to compile because of this.)